features/misc_test.go

```diff
@@ -1,32 +1,31 @@
 package features
 
 import (
-	"fmt"
 	"testing"
 
 	"github.com/cilium/ebpf/internal/testutils"
 )
 
 func TestHaveMisc(t *testing.T) {
-	tests := map[miscType]struct {
+	tests := map[string]struct {
+		typ       miscType
 		probe     func() error
 		minKernel string
 	}{
-		largeInsn:    {probe: HaveLargeInstructions, minKernel: "5.1"},
-		boundedLoops: {probe: HaveBoundedLoops, minKernel: "5.2"},
-		v2ISA:        {probe: HaveV2ISA, minKernel: "4.13"},
-		v3ISA:        {probe: HaveV3ISA, minKernel: "5.0"},
+		"large instructions": {typ: largeInsn, probe: HaveLargeInstructions, minKernel: "5.2"},
```

Comment:
Makes me think the probes should just have been `internal.FeatureTest` in the first place.

Reply:
This can still be moved later on, but I think over time we want to achieve the opposite. We want to expose as many of these as possible to the caller to provide `bpftool feature probe` in Go.